spec/spec_helper.rb

```diff
@@ -15,4 +15,4 @@
   config.include Rack::Test::Methods
 end
 
-Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each { |f| require f }
+Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].sort.each { |f| require f }
```

Comment:
Didn't look into this but it's likely to make it deterministic, which is a good call.